pkgs/top-level/python-packages.nix

```diff
@@ -5375,6 +5377,7 @@ in {
   mkdocs-material-extensions = callPackage ../development/python-modules/mkdocs-material/mkdocs-material-extensions.nix { };
   mkdocs-minify = callPackage ../development/python-modules/mkdocs-minify { };
   mkdocs-redirects = callPackage ../development/python-modules/mkdocs-redirects { };
+  mkdocs-exclude = callPackage ../development/python-modules/mkdocs-exclude { };
```

Comment:
With alphabetical sorting this goes after mkdocs-drawio-exporter.

Reply:
Okay.